kasa/tests/test_common_modules.py

```diff
@@ -89,35 +89,39 @@ async def test_light_effect_module(dev: Device, mocker: MockerFixture):
     assert light_effect_module.has_custom_effects is not None
 
     await light_effect_module.set_effect("Off")
-    assert call.call_count == 1
+    call.assert_called()
     await dev.update()
     assert light_effect_module.effect == "Off"
     assert feat.value == "Off"
+    call.reset_mock()
 
     second_effect = effect_list[1]
     await light_effect_module.set_effect(second_effect)
-    assert call.call_count == 2
+    call.assert_called()
     await dev.update()
```

Comment:
Had to modify this test logic as light_effect requires two calls instead of one.